resources/serverless/charts/k3s-tests/templates/k3s-test.yaml

```diff
@@ -35,8 +35,16 @@ spec:
       containers:
         - name: test
           image: "{{ include "imageurl" (dict "reg" .Values.global.containerRegistry "img" .Values.global.testImages.function_controller_test) }}"
-          args: ["serverless-integration"]
+          args: ["{{ .Values.testSuite }}"]
           env:
+          {{- if ( and (eq .Values.testSuite "git-auth-integration" ) (not (empty .Values.testSuite)) ) }}
+            - name: APP_GITHUB_AUTH_KEY
+              value: "{{ .Values.githubPrivateKey }}"
```

Comment:
I would use: `.Values.gitAuth.github.privateKey` and apply similar to the azure.

Reply:
I think that would just complicate passing the overrides, since we generate them using the test script. I wouldn't do that unless there is really a need.